### DISCUSSION TITLE
Padding around cards

### DISCUSSION THREAD

**Discussion by inari-codes** [beercss/beercss#268] [General]
Hi! I recently tried BeerCSS and it's pretty easy to use for someone who wants to build a nice looking UI fast! Thank you for making this by the way 😁

Though I want to have some padding around the cards, just like in the second image (Material Components Android Catalog), how can I do that and also adding a navigation bar to the bottom because when i tried to add it, it just doesn't show up

![Screenshot_20240219-105037_Chrome](https://github.com/beercss/beercss/assets/85934291/54045955-0c95-4ef9-9ef3-0d800ba9dd7f)

![Screenshot_20240219-105312_M3 Catalog](https://github.com/beercss/beercss/assets/85934291/2407c97f-02e2-40db-941a-a870cf97b7ae)

**Comment by inari-codes** [comment]
[MD3 Site (1).zip](https://github.com/beercss/beercss/files/14326250/MD3.Site.1.zip)
Here's my HTML file if you want to help me

**Comment by leonardorafael** [comment]
Can you provide a codepen example? You can use this pre-configurated example https://codepen.io/leo-bnu/pen/yLwrKXR
